ghdocs/COMPONENTS.md

```diff
@@ -24,6 +24,7 @@ For simple instructions for using Fabric components in Office Add-ins, see the a
 |Search box|![Search box](http://odux.azurewebsites.net/github/img/Search.png)|
 |Toggle|![Toggle](http://odux.azurewebsites.net/github/img/Toggle.png)|
 |Text fields|![Text field](http://odux.azurewebsites.net/github/img/TextFields.png)|
+|Date/time Picker|![Date/time picker](http://i.imgur.com/odFYZjM.png)![Date/time picker small](http://i.imgur.com/k0Yz8Bo.png)
```

Comment:
Please change the component name to "Date Picker" and place it alphabetically in the list.